lib/app_web/endpoint.ex

```diff
@@ -42,6 +42,6 @@ defmodule AppWeb.Endpoint do
 
   plug Plug.MethodOverride
   plug Plug.Head
-  plug Plug.Session, AuthPlug.session_options
+  plug Plug.Session, @session_options
```

Comment:
All I had to do was uncomment the `@session_options` list above and update this line.
The rest of the PR is adding Travis-CI for testing and an `.env_sample` file for clarity.